draft-ietf-jsonpath-base.md

```diff
@@ -1453,7 +1453,9 @@ LCALPHA             = %x61-7A  ; "a".."z"
 function-expr       = function-name "(" S [function-argument
                          *(S "," S function-argument)] S ")"
 function-argument   = literal /
+                      singular-query /
                       filter-query / ; (includes singular-query)
+                      logical-expr / ; NEW, NOT PART OF FIX
```

Comment:
Er, this is part of the fix to allow the likes of `$[?bar(1==1)]`.

Reply:
The text following this ABNF section needs updating to match. Cf. PR #437.

Reply:
I deleted the sentence, as it no longer adds value.